ansible_collections/arista/avd/docs/installation/requirements.md

```diff
@@ -15,7 +15,11 @@ If you leverage [CloudVision](https://www.arista.com/en/products/eos/eos-cloudvi
 
 ## Supported Ansible Versions
 
-- ansible-core from __2.11.3__ to __2.12.x__
+- ansible-core from __2.11.3__ to __2.12.x__ excluding __2.12.0__ to __2.12.5__
+
+Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (fqcn + short name).")
```

Comment:
```suggestion
Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (fqdn + short name).")
```